### ISSUE TITLE
Can not start a new select if all electrodes are unselected in the current select

### ISSUE THREAD

**Issue by leofuturer** [leofuturer/EWOD-GUI2.0#286]
Currently the "shift select" works in a way that by holding the "shift", the user can constantly select and deselect multiple electrodes -> then they can move or perform other operations by right click to open the context menu. However, one scenario to improve is that: when a user selects some electrodes by doing "shift select" -> then unselects those electrodes (so that there's no selected electrode on the canvas), they can not start a new selection by simply doing "left click and drag". They have to hold "shift" and do the "left click and drag".

https://user-images.githubusercontent.com/40692943/219988031-582f3ec4-77a7-43c0-a539-9e84639e2e9a.mp4

This is not intuitive because when there's nothing selected on the canvas, apparently the user can not move or do any other operations. So the only thing they can do (in the "select and move" mode) is to select some electrodes. Naturally, they may expect this is a new selection (without needing to hold the "shift").

To sum up, a better way is: when a user selects some electrodes by doing "shift select" -> then unselects those electrodes (so that there's no selected electrode on the canvas), a new selection should be started so that they can simply do "left click and drag" to select without holding the "shift"
